apps/framework-cli/src/cli.rs

```diff
@@ -168,16 +184,6 @@ async fn top_command_handler(settings: Settings, commands: &Commands) {
                 controller.run_routines(run_mode);
             }
             Commands::Docker { sub_method } => match sub_method.as_str() {
```

Comment:
should we remove this all together?

Reply:
No we still need `moose docker build` and I imagine we'll want `moose docker push` to push to repos later. I believe @03cranec has a linear ticket for that.